src/main/scala/chisel3/util/Bitwise.scala

```diff
@@ -73,7 +103,14 @@ object Reverse {
       Cat(doit(in(half-1,0), half), doit(in(length-1,half), length-half))
     }
   }
+  
   /** Returns the input in bit-reversed order. Useful for little/big-endian conversion.
+    *  
+    * @example {{{
+    * Reverse("b1101".U)  // equivalent to "b1011".U
+    * Reverse("b1101".U(8))  // equivalent to "b10110000".U
```

Comment:
This should be `"b1101".U(8.W)` since due to aliasing with apply we don't allow Ints as arguments

Also, stylistically, I prefer using `.asUInt(width.W)` when giving a width to a literal (and `.U` when not), but that's just my opinion